python/src/puml/gen_puml.aac

```diff
@@ -0,0 +1,131 @@
+plugin:
+  name: Generate PlantUML Diagrams
+  package: aac.plugins.puml
+  description: |
+    An AaC plugin that enables PlantUML diagrams to be generated from AaC
+    definitions.
+  commands:
+    - name: puml-component
+      help_text: Converts an AaC-defined system to a PlantUML component diagram.
+      input:
+        - name: architecture-file
+          type: file
+          description: |
+            A path to a YAML file containing an AaC-defined system from which to
+            generate a PlantUML component diagram.
+        - name: output-directory
+          type: directory
+          description: |
+            The output directory into which the PlantUML (.puml) diagram file
+            will be written.
+      acceptance:
+        name: Component Diagram Test
+        scenarios:
+          - name: Output PlantUML component diagram for valid architecture.
+            given:
+              - The {{puml.input.architecture-file}} contains a valid model.
+            when:
+              - The aac app is run with the puml-component command.
+            then:
+              - A PlantUML component diagram of the model is printed to the console.
```

Comment:
```suggestion
```

Looks like these are left over from initial work on the plugin. According to JD it started as dump to console then moved to file output. Console was dropped at some point, but definition didn't get updated. 

I just tested legacy to see if it would dump to console if I did not provide an output directory and get the following return:

```
(venv) gitpod /workspace/AaC/python/model/alarm_clock (puml-changes-requirements-diagram) $ aac puml-component alarm_clock.yaml 
Usage: aac puml-component [OPTIONS] ARCHITECTURE_FILE OUTPUT_DIRECTORY
Try 'aac puml-component -h' for help.

Error: Missing argument 'OUTPUT_DIRECTORY'.
```

Reply:
changed to "written to the output directory"